packages/monaco/src/browser/monaco-editor.ts

```diff
@@ -117,6 +119,17 @@ export class MonacoEditor implements TextEditor, IEditorReference {
         this.toDispose.push(codeEditor.onDidBlurEditor(() =>
             this.onFocusChangedEmitter.fire(this.isFocused())
         ));
+        this.toDispose.push(codeEditor.onMouseDown(e => {
+            const { lineNumber, column } = e.target.position;
+            const event = {
+                target: {
+                    ...e.target,
```

Comment:
Why to copy other properties if they are not exposed? `range` and `mosueColumn` should not be copied but properly converted from 1-bazed to 0-based positions.